ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -878,6 +879,119 @@ namespace
         return true;
     }
 
+    namespace fft_v7
+    {
+        struct InfoForFFT7
+        {
+            std::vector<float> input_data;
+            std::vector<int64_t> axes_data;
+            Shape input_data_shape;
+            Shape axes_data_shape;
+            Shape output_shape;
+        };
+
+        std::vector<int64_t> canonicalize_axes(const std::vector<int64_t>& axes, int64_t input_rank)
+        {
+            int64_t complex_data_rank = input_rank - 1;
+            std::vector<int64_t> result(axes.begin(), axes.end());
+
+            for (int64_t& axis : result)
+            {
+                if (axis < 0)
+                {
+                    axis += complex_data_rank;
+                }
+            }
+
+            return result;
+        }
```

Comment:
This function duplicates canonicalize_axes() from fft.cpp line 66. Can you use canonicalize_axes from fft.cpp here?

Reply:
Done.